plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -606,6 +606,8 @@ BOOST_FIXTURE_TEST_CASE(bytes_in_flight, http_plugin_test_fixture) {
          req.set(http::field::host, "127.0.0.1:8891");
          boost::beast::http::write(s, req);
       }
+      //make sure got to the point http threads had responses queued
```

Comment:
why does it matter if the response is queued or not at this point?

Reply:
This avoids `boost::beast::http::status::service_unavailable` on my machine

Reply:
maybe after the `boost::beast::http::write(s, req)` try doing a `s.wait(wait_read)`, that will guarantee that the response is in flight. just a guess though